src/app.js

```diff
@@ -3,6 +3,10 @@ const app = express();
 const port = 3000;
 
 app.get("/", (req, res) => {
+  res.send("Uhoh POOPY, HAHA STINKY");
```

Comment:
What the heck is this, this NOT appropriate!!!

Reply:
Suck a fat one bossman